masu/processor/orchestrator.py

```diff
@@ -35,16 +35,40 @@ class Orchestrator():
 
     """
 
-    def __init__(self):
+    def __init__(self, billing_source=None):
         """
         Orchestrator for processing.
 
         Args:
-            None
+            billing_source (String): Individual account to retrieve.
         """
-        self._accounts = AccountsAccessor().get_accounts()
+        self._accounts = self.get_accounts(billing_source)
         self._processing_requests = []
 
+    @staticmethod
+    def get_accounts(billing_source=None):
```

Comment:
It's not clear to me why this belongs in the Orchestrator. Can you help me understand?